src/vaadin-text-field-mixin.html

```diff
@@ -225,6 +226,15 @@
           observer: '_labelChanged'
         },
 
+        /**
+         * String used for the helper text.
```

Comment:
Please add `@type {string | null}` for TS definitions, as we consider `null` supported.

Reply:
Added.